dbt/adapters/trino/column.py

```diff
@@ -22,4 +24,41 @@ def from_description(cls, name: str, raw_data_type: str) -> "Column":
         # some of the Trino data types specify a type and not a precision
         if raw_data_type.startswith(("array", "map", "row")):
             return cls(name, raw_data_type)
-        return super(TrinoColumn, cls).from_description(name, raw_data_type)
+        match = re.match(r"([^(]+)(\([^)]+\))?([\w ]+)?", raw_data_type)
+        if match is None:
+            raise RuntimeException(f'Could not interpret data type "{raw_data_type}"')
+        data_type, size_info, data_type_2nd_part = match.groups()
```

Comment:
Rename `data_type_2nd_part` to `data_type_suffix`

Reply:
changed